### PR TITLE
planner, executor: return err when INSERT/UPDATE/ANALYZE/DELETE a view

### PR DESCRIPTION
**Pull request by XuHuaiyu**
### What problem does this PR solve? 
TiDB does not support INSERT/ UPDATE/ ANALYZE/ DELETE a view now, we return error instead panic after this commit.

### What is changed and how it works?
Check when tableInfo.IsView when meeting INSERT/ UPDATE/ ANALYZE a view.

### Check List 

Tests 

 - Integration test

Code changes

 - Has exported function/method change

Side effects

N/A

Related changes

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8848)

### CODE REVIEW

File: planner/core/logical_plan_builder.go [hunk 1]
```diff
@@ -2216,6 +2216,9 @@ func (b *PlanBuilder) buildUpdate(update *ast.UpdateStmt) (Plan, error) {
 		if dbName == "" {
 			dbName = b.ctx.GetSessionVars().CurrentDB
 		}
+		if t.TableInfo.IsView() {
+			return nil, errors.New(fmt.Sprintf("update view %s is not supported now.", t.Name.O))
+		}
 		b.visitInfo = appendVisitInfo(b.visitInfo, mysql.SelectPriv, dbName, t.Name.L, "")
```
Comment (eurekaka) [inline planner/core/logical_plan_builder.go]: Shall we raise error for `Delete` as well?

**Comment by zz-jason** [comment]
@XuHuaiyu any update?

File: planner/core/planbuilder.go [hunk 2]
```diff
@@ -688,6 +688,9 @@ func getPhysicalIDs(tblInfo *model.TableInfo, partitionNames []model.CIStr) ([]i
 func (b *PlanBuilder) buildAnalyzeTable(as *ast.AnalyzeTableStmt) (Plan, error) {
 	p := &Analyze{MaxNumBuckets: as.MaxNumBuckets}
 	for _, tbl := range as.TableNames {
+		if tbl.TableInfo.IsView() {
+			return nil, errors.New(fmt.Sprintf("analyze %s is not supported now.", tbl.Name.O))
```
Comment (lonng) [inline planner/core/planbuilder.go]: Use `errors.Errorf` to make the style more uniform. See: L1086/L1088

**Comment by lonng** [comment]
LGTM

**Comment by XuHuaiyu** [comment]
PTAL @eurekaka

**Comment by zz-jason** [comment]
@XuHuaiyu Please merge master and fix ci.

**Pull request merged**